frontend/dialob-material-dashboard/README.md

```diff
@@ -0,0 +1,149 @@
+# Dialob Material Dashboard
+This package provides a Dialob Admin UI View which enables users to easily interact with the Dialob forms.
+
+## Install
+Installation depends on package manager you are using.
+
+```sh
+yarn add @dialob/dashboard-material 
+```
+or
+```sh
+pnpm add @dialob/dashboard-material 
+```
+
+## Quick-start
+```jsx
+import React from 'react';
+import { ThemeProvider } from '@mui/material/styles';
+import { DialobAdmin, DialobAdminConfig } from "@dialob/dashboard-material";
+
+const config: DialobAdminConfig = {
+	csrf : {
+		key : 'X-CSRF-TOKEN',													 
+		value : '6cf7b545-6d09-4a95-b8ad-85afad13af7c'
+	},
+	dialobApiUrl: 'http://localhost:8085/dialob',
+	setLoginRequired: cfg.setLoginRequired,
+	setTechnicalError: cfg.setTechnicalError,
+	language: "en"
+}
+
+const App = () => {
+ return (
+	  <ThemeProvider theme={yourTheme}>
+			<DialobAdmin 
+				config={config}
+			/>
+		</ThemeProvider>
+ );
+}
+```
+
+## Local Development Environment
+In order to view this component localy without embedding this module into another application,
+you can run in the root directory:
+
+```sh
+pnpm dev
+```
+
+And then you can change themes on the fly and see how the component behaves.
+
+## Interfaces
+
+```jsx
+interface CsrfShape {
+	'key': string,
+	'value': string
+}
+
+interface DialobAdminConfig {
+	dialobApiUrl: string; // base url for Dialob Api
+	setLoginRequired: () => void; // used by POP, but can be set as a function that does nothing if not needed
+	setTechnicalError: () => void; // used by POP, but can be set as a function that does nothing if not needed
+	language: string; // Current locale used in your application
+	csrf: CsrfShape; // Adjust according to your application csrf settings
+}
+
+interface DialobAdminViewProps {
+	config: DialobAdminConfig;
+	showSnackbar?: (message: string, severity: 'success' | 'error') => void;
```

Comment:
Please rename `showSnackbar` -> `showNotification` (It doesn't depend on what type of notification is used)